instance/tests/test_github.py

```diff
@@ -93,7 +93,7 @@ def test_get_pr_by_number(self):
             '**Description**\r\n\r\nHello!\nDesc with unicode «ταБЬℓσ»\r\n'
             '- - -\r\n**Settings**\r\n```yaml\r\nEDXAPP_FEATURES:\r\n  ALLOW: true\r\n```')
         self.assertEqual(pr.number, 8474)
-        self.assertEqual(pr.fork_name, 'open-craft/edx-platform')
+        self.assertEqual(pr.fork_name, 'edx/edx-platform')
```

Comment:
That's not correct - the PR code is on the origin repository (ie for a PR from open-craft/edx-platform to edx/edx-platform, the code resides on open-craft/edx-platform), the task relies on this to point to the right code to deploy. It's only the PR URL that is on the destination repository (edx/edx-platform).